code/modules/clothing/masks/_masks.dm

```diff
@@ -5,6 +5,7 @@
 	slot_flags = ITEM_SLOT_MASK
 	strip_delay = 40
 	equip_delay_other = 40
+	equip_delay_self = 0.6 SECONDS
```

Comment:
This nerfs internals. Nitrous flood memes say hi.

Reply:
of all people he's not going to care

Reply:
is less than a second

Reply:
> 
> 
> of all people he's not going to care

HE CAN'T KEEP GETTING AWAY WITH IT!!

Reply:
![sgQPw7n-Imgur](https://user-images.githubusercontent.com/42839747/114761507-d9b6f700-9d60-11eb-9d19-9982d0b75722.gif)